tests/test_io.py

```diff
@@ -30,8 +30,21 @@ def test_load_program_texts_01(self):
         print("current work directory: ", os.getcwd())
         assert dfastmi.io.load_program_texts("dfastmi/messages.UK.ini") == None
 
+    def test_load_program_texts_02(self):
+        """
+        Testing load_program_texts.
+        """
+        print("current work directory: ", os.getcwd())
+        with pytest.raises(Exception) as cm:
+            dfastmi.io.load_program_texts("tests/files/messages.duplicate_keys.ini")
+        assert str(cm.value) == 'Duplicate entry for "checksum" in "tests/files/messages.duplicate_keys.ini".'
+
+@pytest.fixture
```

Comment:
you can move this up

Reply:
will do in https://issuetracker.deltares.nl/browse/DFAST-86